site/docs/tools/sql.md

```diff
@@ -127,3 +127,11 @@ Merging branch `dev` into `base` for the `nessie` catalog can be done via:
 * `MERGE BRANCH dev INTO base IN nessie`
 
 Note that in case `base` doesn't exist, Nessie will fall back to the default branch (`main`).
+
+!!! note
```

Comment:
What's the relation between compaction and merge?

Reply:
Moved under a new section called "Usage with Iceberg CALL procedures"